renderer-macos/lib/stubs/BriskButton.c

```diff
@@ -10,29 +13,40 @@
   value onClick;
 }
 
-@synthesize attributedString;
-@synthesize attributedProps;
-@synthesize paragraphStyle;
+@synthesize brisk_paragraphStyle;
 
 - (id)init {
   self = [super init];
 
   if (self) {
-    self.attributedString = [[NSMutableAttributedString alloc] init];
-    self.attributedProps = [NSMutableDictionary dictionary];
-    self.paragraphStyle =
+    self.brisk_attributedString = [[NSMutableAttributedString alloc] init];
+    self.brisk_textAttributes = [NSMutableDictionary dictionary];
+    self.brisk_paragraphStyle =
         [[NSParagraphStyle defaultParagraphStyle] mutableCopy];
 
-    self.attributedProps[NSParagraphStyleAttributeName] = self.paragraphStyle;
+    self.brisk_textAttributes[NSParagraphStyleAttributeName] = self.brisk_paragraphStyle;
   }
 
   return self;
 }
 
-- (void)applyTextStyle {
-  NSRange range = NSMakeRange(0, self.attributedString.length);
-  [self.attributedString setAttributes:self.attributedProps range:range];
-  [self setAttributedTitle:self.attributedString];
+#pragma mark - BriskStylableText
+
+- (void)brisk_setLineBreakMode:(NSLineBreakMode)mode {
+  self.lineBreakMode = mode;
+}
+
+- (void)brisk_addAttribute:(NSAttributedStringKey)key value:(id)value {
+  [self.brisk_attributedString 
+    addAttribute:key
+    value:value 
+    range:NSMakeRange(0, self.brisk_attributedString.length)];
+}
```

Comment:
This doesn't look reformatted, can you make a pass with `clangformat`? I could add a `.clang-format` file but currently I'm just using default settings in VSCode.